src/Components/Layout/ResponsiveGrid.tsx

```diff
@@ -1,15 +1,18 @@
 import Box from '@mui/material/Box';
 import Grid from '@mui/material/Grid';
-import Paper from '@mui/material/Paper';
-import { experimentalStyled as styled } from '@mui/material/styles';
 
-const Item = styled(Paper)(({ theme }) => ({
-  backgroundColor: theme.palette.mode === 'dark' ? '#1A2027' : '#fff',
-  ...theme.typography.body2,
-  padding: theme.spacing(2),
-  textAlign: 'center',
-  color: theme.palette.text.secondary,
-}));
+import ServiceCard from '../Organisms/ServiceCard/ServiceCard';
+
+// import Paper from '@mui/material/Paper';
```

Comment:
is that comment important? could be removed?

Reply:
Thx for the comment. Removed.